public_dropin_gpu_environments/triton_server/dr_requirements.txt

```diff
@@ -1,2 +1,140 @@
+#
+# This file is autogenerated by pip-compile with Python 3.9
+# by the following command:
+#
+#    pip-compile --index-url=https://artifactory.devinfra.drdev.io/artifactory/api/pypi/pypi/simple/ --no-emit-index-url --no-emit-trusted-host dr_requirements.in
```

Comment:
We must not expose our internal artifactory URL.

Reply:
removed